### PR TITLE
Add support for the Kinto Account plugin.

### LINKED ISSUE #436: Add support for accounts authentication mecanism.

### PR DESCRIPTION
**Pull request by Natim**
Fixes #436

### CODE REVIEW

File: src/components/AuthForm.js [hunk 1]
```diff
@@ -103,6 +103,36 @@ const baseUISchema = {
 };
 
 const authSchemas = {
+  account: {
```
Comment (n1k0) [inline src/components/AuthForm.js]: Just me or this `account` schema is basically just the `account` one with its title modified? If so we should rather create a common base and extend it appropriately.

File: src/actions/session.js [hunk 2]
```diff
@@ -100,8 +100,9 @@ export function logout(): {
 }
 
 function navigateToFxA(server: string, redirect: string): NavigationResult {
-  document.location
-    .href = `${server}/fxa-oauth/login?redirect=${encodeURIComponent(redirect)}`;
+  document.location.href = `${server}/fxa-oauth/login?redirect=${encodeURIComponent(
+    redirect
+  )}`;
```
Comment (n1k0) [inline src/actions/session.js]: I'm curious as to why this has been reformatted. Did you `npm install` to retrieve the latest version of *prettier*?

Comment (Natim) [inline src/actions/session.js]: Yes

File: src/components/AuthForm.js (hunk 1 above)
Comment (Natim) [inline src/components/AuthForm.js]: It is indeed.

File: src/actions/session.js (hunk 2 above)
Comment (Natim) [inline src/actions/session.js]: do you want me to revert this specific change?

Comment (n1k0) [inline src/actions/session.js]: No it's ok

**Comment by n1k0** [comment]
Seems we have an issue with the prettier version you're using when running `cs-format` on your machine, as per travis job result:  

```
> kinto-admin@1.13.3 cs-check /home/travis/build/Kinto/kinto-admin
> prettier-check $npm_package_prettierOptions '{src,test}/**/*.js'
Forgot to run prettier? There are files without correct code style:
src/actions/session.js
src/components/AuthForm.js
src/components/collection/CollectionForm.js
src/components/HistoryTable.js
src/components/HomePage.js
src/components/Notifications.js
src/components/Sidebar.js
src/components/TagsField.js
src/containers/collection/CollectionAttributesPage.js
src/containers/collection/CollectionPermissionsPage.js
src/permission.js
src/routes.js
src/utils.js
test/components/CollectionRecords_test.js
```

Could you please ensure you're running the exact same version required in `package.json`?

Also it seems a test is now breaking following a recent react upgrade:

```
1) App component Session top bar should not render a session top bar when not authenticated:
     Error: Warning: Failed prop type: The prop `params` is marked as required in `t`, but its value is `undefined`.
    in t (created by App)
    in App
      at Console.<anonymous> (test/test_utils.js:17:11)
 ```

Could you please fix this one as well so we have a green build?

Thanks!

**Comment by Natim** [comment]
Yes sure, thanks for the review.

**Comment by Natim** [comment]
I use:

- prettier@1.4.4 
- prettier-check@1.0.0 

I have no idea looking at the package.json file if it is good or not.

**Comment by Natim** [comment]
Rebase with prettier 1.0.0

**Pull request merged**